x-pack/plugins/ml/public/components/json_tooltip/tooltips.js

```diff
@@ -0,0 +1,315 @@
+      defaultMessage: 'Unique identifier for job, can use lowercase alphanumeric and underscores.'
+    })
+  },
+  new_job_description: {
+    text: i18n.translate('xpack.ml.jsonTooltip.newJobDescriptionTooltip', {
+      defaultMessage: 'Optional descriptive text.'
+    })
+  },
+  new_job_group: {
+    text: i18n.translate('xpack.ml.jsonTooltip.newJobGroupTooltip', {
+      defaultMessage: 'Optional grouping for jobs. New groups can be created or picked from the list of existing groups.'
+    })
+  },
+  new_job_custom_urls: {
+    text: i18n.translate('xpack.ml.jsonTooltip.newJobCustomUrlsTooltip', {
+      defaultMessage: 'Optional drill-through links to source data. Supports string substitution for analyzed fields e.g. $hostname$.'
+    })
+  },
+  new_job_bucketspan: {
+    text: i18n.translate('xpack.ml.jsonTooltip.newJobBucketspanTooltip', {
+      defaultMessage: 'Interval for time series analysis.'
+    })
+  },
+  new_job_sparsedata: {
+    text: i18n.translate('xpack.ml.jsonTooltip.newJobSparsedataTooltip', {
+      defaultMessage: 'Check if you wish to ignore empty buckets from being considered anomalous.'
+    })
+  },
+  new_job_summarycountfieldname: {
+    text: i18n.translate('xpack.ml.jsonTooltip.newJobSummarycountfieldnameTooltip', {
+      defaultMessage: 'Optional, for use if input data has been pre-summarized e.g. doc_count.'
+    })
+  },
+  new_job_categorizationfieldname: {
+    text: i18n.translate('xpack.ml.jsonTooltip.newJobCategorizationfieldnameTooltip', {
+      defaultMessage: 'Optional, for use if analyzing unstructured log data. Using text data types is recommended.'
+    })
+  },
+  new_job_categorizationfilters: {
+    text: i18n.translate('xpack.ml.jsonTooltip.newJobCategorizationfiltersTooltip', {
+      defaultMessage: 'Optional, apply regular expressions to the categorization field'
+    })
+  },
+  new_job_detectors: {
+    text: i18n.translate('xpack.ml.jsonTooltip.newJobDetectorsTooltip', {
+      defaultMessage: 'Defines the fields and functions used for analysis.'
+    })
+  },
+  new_job_influencers: {
+    text: i18n.translate('xpack.ml.jsonTooltip.newJobInfluencersTooltip', {
+      defaultMessage: 'Select which categorical fields have influence on the results. \
+      Who/what might you \"blame\" for an anomaly? Recommend 1-3 influencers.'
+    })
+  },
+  new_job_detector_description: {
+    text: i18n.translate('xpack.ml.jsonTooltip.newJobDetectorDescriptionTooltip', {
+      defaultMessage: 'User-friendly text used for dashboards.'
+    })
+  },
+  new_job_detector_function: {
+    text: i18n.translate('xpack.ml.jsonTooltip.newJobDetectorFunctionTooltip', {
+      defaultMessage: 'Analysis functions to be performed e.g. sum, count.'
+    })
+  },
+  new_job_detector_fieldname: {
+    text: i18n.translate('xpack.ml.jsonTooltip.newJobDetectorFieldnameTooltip', {
+      defaultMessage: 'Required for functions: sum, mean, median, max, min, info_content, distinct_count.'
```

Comment:
@peteharverson I believe we also need to move `sum, mean, median, max, min, info_content, distinct_count` to values